packages/cli-raster/src/cogify/covering/tile.cover.ts

```diff
@@ -129,11 +130,17 @@ export async function createTileCover(ctx: TileCoverContext): Promise<TileCoverR
 
     // Scale the tile bounds slightly to ensure we get all relevant imagery
     const scaledBounds = bounds.scaleFromCenter(1.05);
-    const tileBounds = Projection.get(ctx.tileMatrix).projectMultipolygon(
```

Comment:
Small fix in the tile cover to split multipolygons if crossing antimerdian into wsg84 then convert to target bounds in order to find the source tifs on the other side of antimerdian.